crop.mjs

```diff
@@ -48,37 +48,59 @@ const lineNames = {
 		'miserable',
 		'furious',
 		null,
+		'afraid',
 	],
+	basil: [
+		'neutral',
+		'happy',
+		'ecstatic',
+		'afraid',
+		'sad',
+		'depressed',
+		'angry',
+		'enraged',
+		'defeated',
+		'injured',
+		'victory',
+		'manic',
+		'miserable',
+		'furious',
+	]
 };
 
-readdirSync('spritesheets/characters').forEach(async (file) => {
-	const image = sharp(`spritesheets/characters/${file}`);
+readdirSync('./public/spritesheets/characters').forEach(async (file) => {
+	const image = sharp(`./public/spritesheets/characters/${file}`);
 	const metadata = await image.metadata();
 	const lines =
 		file === 'sunny.png'
 			? {
-					count: 15,
+					count: lineNames.sunny.length,
 					names: lineNames.sunny,
 			  }
 			: file === 'omori.png'
 			? {
-					count: 15,
+					count: lineNames.omori.length,
 					names: lineNames.omori,
 			  }
+			: file == "basil.png"
```

Comment:
Could you please change this from `"basil.png"` to `'basil.png'`?

Reply:
Done!